frontend/src/views/ShareSecretPublicPage/components/SecretTable.tsx

```diff
@@ -10,39 +11,68 @@ type Props = {
   copyUrlToClipboard: () => void;
 };
 
+const replaceContentWithDot = (str: string) => {
```

Comment:
I remember @akhilmhdh had to fiddle with this quite a bit for secrets value in the dashbaord. Tagging Akhi so he can double check